docs/developer/style.md

```diff
@@ -0,0 +1,213 @@
+
+# Go Style Guide in Vald
+
+## Table of Contents
+
+- [Introduction](#Introduction)
+- [Style](#Style)
+- [Test](#Test)
+	- [Table-Driven-Test](#Table-Driven-Test)
+
+## Introduction
+
+## Style
+
+## Test
+
+This section will describe the rule for writing test code in Vald.
+
+### Table-Driven-Test
+
+Use table-driven tests with subtests to avoid duplicating code. 
+
+<table>
+<thead><tr><th>Bad</th><th>Good</th></tr></thead>
```

Comment:
Which is better the left column is `Bad` or `Good`?:thinking:
How do you think @kevindiu ?

Reply:
I think `Bad` on the left column is better because I think it is better to let user to understand the `Bad` case first and then compare with `Good`, but if we write the `Good` on the left column user may avoid the `Bad` case.